tests/lib/ignored-paths.js

```diff
@@ -164,6 +164,30 @@ describe("IgnoredPaths", () => {
                 )
             );
         });
+
+
+    });
+
+    describe("caching file reads", () => {
+
+        let readFileSyncCount;
+
+        before(() => {
+            readFileSyncCount = sinon.spy(fs, "readFileSync");
```

Comment:
Has a sinon sandbox been defined elsewhere in scope? If so, I would prefer we use that- otherwise this LGTM.